citylearn/electric_vehicle.py

```diff
@@ -0,0 +1,493 @@
+import inspect
+import math
+from typing import List, Mapping, Tuple, Union, Dict
+from gym import spaces
+import numpy as np
+from citylearn.base import Environment, EpisodeTracker
+from citylearn.data import EnergySimulation, CarbonIntensity, Pricing, Weather, EVSimulation
+from citylearn.energy_model import Battery, ElectricHeater, HeatPump, PV, StorageTank
+from citylearn.preprocessing import Normalize, PeriodicNormalization
+import random
+import copy
+
+class electric_vehicle(Environment):
+
+    def __init__(self, ev_simulation: EVSimulation,episode_tracker: EpisodeTracker, observation_metadata: Mapping[str, bool],
+                 action_metadata: Mapping[str, bool], battery: Battery = None, auxBattery: Battery = None, min_battery_soc: int = 20,
```

Comment:
`auxBattery` should be spelled in full `auxilliary_battery`.

Reply:
Removed auxBattery altogether.